prbt_hardware_support/launch/modbus_client.launch

```diff
@@ -17,19 +17,20 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 <launch>
 
-<!-- default modbus server ip (PSS400) and port number -->
-<arg name="modbus_server_ip" default="192.168.0.10" />
-<arg name="modbus_server_port" default="502" />
+  <!-- default modbus server ip (PSS400) and port number -->
+  <arg name="modbus_server_ip" default="192.168.0.10" />
+  <arg name="modbus_server_port" default="502" />
 
-<!-- the register range for reading is determined automatically from the api spec,
-      however this can be overruled by specifying the following arguments -->
-<arg name="index_of_first_register_to_read" default="" />
-<arg name="num_registers_to_read" default="" />
+  <!-- the register range for reading is determined automatically from the api spec,
+       however this can be overruled by specifying the following arguments -->
+  <arg name="index_of_first_register_to_read" default="" />
+  <arg name="num_registers_to_read" default="" />
 
-<!-- define the connected safety hardware -->
-<arg name="safety_hw" default="pss4000" />
+  <!-- define the connected safety hardware -->
+  <arg name="safety_hw" default="pss4000" />
+  <arg name="read_api_spec_file" default="$(find prbt_hardware_support)/config/modbus_read_api_spec_$(arg safety_hw).yaml" />
 
-<rosparam ns="/prbt/read_api_spec" command="load" file="$(find prbt_hardware_support)/config/modbus_read_api_spec_$(arg safety_hw).yaml" />
+  <rosparam ns="/prbt/read_api_spec" command="load" file="$(arg read_api_spec_file)" />
```

Comment:
then this line would be redundant as well??

Reply:
I think it is not redundant because we mention `modbus_client.launch` as a standalone launch file in the ReadMe.